jax_moseq/models/arhmm/gibbs.py

```diff
@@ -47,11 +47,17 @@ def resample_discrete_stateseqs(seed, x, mask, Ab, Q, pi, **kwargs):
         Discrete state sequences.
     """
     nlags = get_nlags(Ab)
+
+    # TODO Why not use jax.vmap here? Can specify out_axes=-1 to remove
```

Comment:
`jax.lax.map` is used to reduce peak VRAM usage, i.e. to prevent OOM errors for larger datasets.

Reply:
Great to know!